packages/calcite-components/src/components/tile/tile.tsx

```diff
@@ -36,6 +31,11 @@ export class Tile implements ConditionalSlotComponent, InteractiveComponent {
    */
   @Prop({ reflect: true }) active = false;
 
+  /**
+   * Specifies the alignment of the Tile's content.
+   */
+  @Prop({ reflect: true }) alignment: Exclude<Alignment, "end"> = "start";
```

Comment:
Elsewhere we use portions of interface definitions, we use `Extract<>` vs. `Exclude<>` - no preference here except that we are consistent - does one have benefits over the other?

Reply:
Good question, not sure.